ads/_config.js

```diff
@@ -358,7 +358,7 @@ export const adConfig = {
     renderStartImplemented: true,
   },
 
-  'chargeads': {},
+  'chargeads': {},  # Deprecated, to be removed on 2019-04-06
```

Comment:
Thank you for updating the date. 
not python : ) `//` to add comment 😄

Reply:
Sorry :man_facepalming: